### ISSUE TITLE
Absolete API

### ISSUE THREAD

**Issue by VREMSoftwareDevelopment** [arturdm/jacoco-android-gradle-plugin#58]
Using latest Android Studio 3.3

```
WARNING: API 'variant.getJavaCompile()' is obsolete and has been replaced with 'variant.getJavaCompileProvider()'.
	It will be removed at the end of 2019.
	For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
	To determine what is calling variant.getJavaCompile(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.

```

**Comment by peterbetos** [comment]
Pull request is already in place. Not sure who will release this though:
https://github.com/arturdm/jacoco-android-gradle-plugin/pull/53

**Comment by arturdm** [comment]
Fixed in 0.1.4
